### ISSUE TITLE
[Exploit] Permanently immune in Battlegrounds as a paladin

### ISSUE THREAD

**Issue by Wattiqelumes** [tobmaps/TrinityCore#179]
http://www.deathsoft.com/forum/index.php?showtopic=17258

**Comment by tobmaps** [comment]
is it works at latest or no?

**Comment by Wattiqelumes** [comment]
Still bugged =(

**Comment by tobmaps** [comment]
okay, I'm reproduced this, really strange thing. Need to check how it works at retail

**Comment by Wattiqelumes** [comment]
I haven't tried this on arena, but this would pretty much be a free win..

**Comment by tobmaps** [comment]
Think you can't use this on arena

**Comment by Wattiqelumes** [comment]
I'm not able to test right now, can you test it?
If not, I'll write it down and I will test it later this evening. :)

**Comment by Anonimo17** [comment]
lol

**Comment by tobmaps** [comment]
yep, it's happens because there own Issues numeration in my repo
